src/PKSim.Infrastructure/Reporting/Summary/ObserverSetReportBuilder.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentNHibernate.Utils;
+using PKSim.Core.Model;
+using PKSim.Core.Reporting;
+
+namespace PKSim.Infrastructure.Reporting.Summary
+{
+   public class ObserverSetReportBuilder : ReportBuilder<ObserverSet>
+   {
+      protected override void FillUpReport(ObserverSet observerSet, ReportPart reportPart)
+      {
+         reportPart.Title = observerSet.Name;
```

Comment:
Not sure what to put for observer set tooltip. For now, name of each observer in set